lib/cc/cli/init.rb

```diff
@@ -65,6 +64,16 @@ def exclude_paths(paths)
         expanded_paths
       end
 
+      def build_engine_config(engine_config)
+        config = { "enabled" => true }
+
+        if engine_config["default_config"]
+          config["config"] = engine_config["default_config"]
+        end
+
+        config
```

Comment:
I agree. I just used what it was called in the calling method. I think it would be nice to refactor the entire file.